tests/virt/cluster/aaq/test_arq.py

```diff
@@ -221,8 +223,16 @@ def test_arq_memory_hotplug(
         hotplugged_resource,
     ):
         vmi_spec_domain = hotplug_vm_for_aaq_test.vmi.instance.spec.domain
+        samples = TimeoutSampler(
+            wait_timeout=TIMEOUT_1SEC, sleep=TIMEOUT_5SEC, func=lambda: application_aware_resource_quota.instance
```

Comment:
wait_timeout should be at least 5 seconds, sleep - 1 second
and not clear why you are waiting for instance here - it always present, so you can get `application_aware_resource_quota.instance.status.used`

Reply:
probably, you can use `func=check_arq_status_values`  with necessary arguments here

Reply:
sampler shouldn't be part of test body at all.
you can modify `check_arq_status_values` to have it inside (for e.g. instead of `current` param, get `application_aware_resource_quota` and then wait for final status)